spyder/api/widgets/__init__.py

```diff
@@ -981,10 +722,7 @@ def create_window(self):
         self._update_actions()
         window.show()
 
-        # Signals
-        # TODO:
-        # self.sig_refresh_requested.emit()
-
+    # FIXME: Check what changed in this method
```

Comment:
Please address this before merging this PR.

Reply:
Fixed